spec/active_record/connection_adapters/oracle_enhanced_data_types_spec.rb

```diff
@@ -882,6 +882,234 @@ class ::TestSerializeEmployee < ActiveRecord::Base
   end
 end
 
+describe "OracleEnhancedAdapter handling of NCLOB columns" do
+  include SchemaSpecHelper
+
+  before(:all) do
+    ActiveRecord::Base.establish_connection(CONNECTION_PARAMS)
+    @conn = ActiveRecord::Base.connection
+    schema_define do
+      create_table :test_employees, force: true do |t|
+        t.string  :first_name, limit: 20
+        t.string  :last_name, limit: 25
+        t.column  :comments, :ntext
```

Comment:
It does look better. I was getting error and had not updated schema_definitions. Thank you for your help!